chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -62,6 +62,9 @@
 # calculate pg_num based on documentation
 # http://ceph.com/docs/master/rados/operations/placement-groups/
 case num_osds
+when 0
+  Chef::Log.warn("Ceph recipe invoked but there are no OSDs!")
+  return
```

Comment:
With the change to the search criteria, it shouldn't be possible to hit the zero case anymore, but if it somehow *does* come up zero, I thought bailing out somewhat cleanly was less worse than hitting a divide by zero error in the else case.